rules/Php55/Rector/String_/StringClassNameToClassConstantRector.php

```diff
@@ -94,6 +97,15 @@ public function refactor(Node $node): ?Node
     {
         $classLikeName = $node->value;
 
+        $uses = $this->useImportsResolver->resolveForNode($node);
+        foreach ($uses as $use) {
+            $_classLikeName = $this->nodeNameResolver->getName($use) ?? '';
```

Comment:
You should can loop the `$use->uses` and check if the alias property is not Identifier, check the $name->getLast() to ensure on end after the backslash

Reply:
also, the valid use case probably only on targetEntity inside attribute, otherwise, it will be false positive. You may need to create something "SpecialStringToClassConstant" service which handle only on it.